src/main/java/chess/pieces/Color.java

```diff
@@ -0,0 +1,15 @@
+package chess.pieces;
+
+public enum Color {
+    WHITE('p'), BLACK('P');
```

Comment:
`enum`을 선언할때는 한 줄에 하나씩 선언하는 것을 원칙으로 해 주세요.

```suggestion
    WHITE('p'),
    BLACK('P');

Reply:
네에 수정 완료했습니다!